eng/psakefile.ps1

```diff
@@ -74,9 +74,31 @@ Task parser -Depends init {
 }
 
 Task pyqir-tests -Depends init {
-    Invoke-LoggedCommand -workingDirectory (Join-Path $repo.root pyqir-tests) {
-        & $python -m pip install tox
-        & $python -m tox -e test
+    $srcPath = $repo.root
+    $project = "pyqir-tests"
+    $installationDirectory = Resolve-InstallationDirectory
+
+    if (Test-RunInContainer) {
+        Build-ContainerImage $srcPath
+        Write-BuildLog "Running container image:"
+        $ioVolume = "$($srcPath):/io"
+        $llvmVolume = "$($installationDirectory):/usr/lib/llvm"
```

Comment:
The path separator `:` looks OS-specific. I'm not sure if it is supported on Win, if ever run on Win.

Reply:
This code only executes on linux for manylinux support gated by `Test-RunInContainer` above.